R/nm-mode.R

```diff
@@ -86,11 +86,12 @@ any_nm_vars <- function(x) {
   list(found_any = length(ans) > 0, match = ans)
 }
 
-audit_nm_vars <- function(x, param, init, build, nmv, env) {
-  bad_param <- any_nm_vars(names(param))
-  bad_init <- any_nm_vars(names(init))
+audit_nm_vars <- function(spec, x, build, nmv, env) {
+  cmt <- Cmt(x)
+  bad_param <- any_nm_vars(Pars(x))
+  bad_init <- any_nm_vars(cmt)
   bad_cpp <- any_nm_vars(build[["cpp_variables"]][["var"]])
-  audit_dadt <- isTRUE(env[["audit_dadt"]]) && length(nmv[["dcmtn"]]) > 0
+  audit_dadt <- isTRUE(env[["audit_dadt"]])
```

Comment:
Part of the audit unification; we audit with or with `nm-vars`, as long as user didn't say not to audit via block option.